mechanics/micromechanics/tests/test_potentials.cpp

```diff
@@ -0,0 +1,332 @@
+#include <cmath>
+#include <memory>
+
+#include <common/base_agent_data.h>
+#include <gtest/gtest.h>
+
+#include "micromechanics/agent_container.h"
+#include "micromechanics/agent_data.h"
+#include "micromechanics/environment.h"
+#include "micromechanics/potential_interface.h"
+#include "micromechanics/simulation_parameters.h"
+
+// Include internal potential implementations for testing
+#include "potentials/kelvin_voigt_potential.h"
+#include "potentials/morse_potential.h"
+#include "potentials/standard_potential.h"
+
+using namespace physicore;
+using namespace physicore::mechanics::micromechanics;
+using namespace physicore::mechanics::micromechanics::kernels::openmp_solver;
+
+class PotentialTest : public ::testing::Test
```

Comment:
since potentials implementation is in the kernel, this test should be in the kernel's tests. also, I would appreciate the _exact_ test, not just GT and LT. do some simple examples, compute the forces on the paper by hand and then compare your implementation matches expected output